tests/testthat/test-write-tsv.R

```diff
@@ -0,0 +1,36 @@
+context("write_tsv")
+
+test_that("strings are only quoted if needed", {
```

Comment:
Given that `write_tsv()` just calls `write_delim()`, I don't think adding specific tests adds anything. I'd lump together all the writer functions in one set of tests - I think the main thing to check is the `write_delim()` correctly round trips regardless of the delimiter used.